volume.go

```diff
@@ -125,8 +126,9 @@ func (c *Client) NewVolume(v *VolumeConfig) (*VolumeResult, error) {
 // ResizeVolume resizes a volume
 // https://www.civo.com/api/volumes#resizing-a-volume
 func (c *Client) ResizeVolume(id string, size int) (*SimpleResponse, error) {
-	resp, err := c.SendPutRequest(fmt.Sprintf("/v2/volumes/%s/resize", id), map[string]int{
-		"size_gb": size,
+	resp, err := c.SendPutRequest(fmt.Sprintf("/v2/volumes/%s/resize", id), map[string]string{
+		"size_gb": strconv.Itoa(size),
```

Comment:
API is going to accept this as an integer

Reply:
update